### PR TITLE
Feature/fft

### PR DESCRIPTION
**Pull request by quantumshiro**
The branch name is ‘fft’, but it implements the signature scheme falcon.
This library should work.

### CODE REVIEW

**Comment by quantumshiro** [comment]
Oops, I wrote a simple test and it wasn't working.

**Comment by iokira** [comment]
this?

```
error[E0658]: use of unstable library feature 'int_roundings'
   --> src/cryptography/falcon/math.rs:112:14
    |
112 |             .next_multiple_of(8) as usize
    |              ^^^^^^^^^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
```

File: src/cryptography/falcon.rs [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+pub mod field;
+pub mod fft;
+pub mod u32_field;
+pub mod sample;
+pub mod math;
+pub mod encoding;
+pub mod falcon;
+pub mod sampling;
+pub mod falcon512;
+pub mod falcon1024;
```
Comment (iokira) [inline src/cryptography/falcon.rs:13]: The file should end with a NEW LINE.

```suggestion
pub mod falcon1024;

```

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Comment (quantumshiro) [reply src/cryptography/falcon.rs]: I solved this

**Comment by quantumshiro** [comment]
I will investigate the bug once

**Pull request closed without merging**